conf/mesos-site.xml

```diff
@@ -49,7 +49,7 @@
 
   <property>
     <name>mesos.hdfs.cluster.name</name>
```

Comment:
This will become more important when we support multiple HDFS clusters running on the same Mesos cluster, at which point they will use the cluster/framework name as a key in ZK for fetching the frameworkId. This use case is arguably more important than the framework's display name in Mesos.